src/state/slices/limitOrderSlice/selectors.ts

```diff
@@ -193,7 +193,7 @@ export const selectConfirmedLimitOrder = createSelector(
 export const selectLimitOrderSubmissionMetadata = createDeepEqualOutputSelector(
   selectLimitOrderSlice,
   selectQuoteIdParamFromRequiredFilter,
-  (limitOrders, quoteId) => {
+  (limitOrders, quoteId): LimitOrderSubmissionMetadata | undefined => {
```

Comment:
The guts of the undefined checks across this diff. This was incorrect, as `limitOrders.orderSubmission` is a `Record<QuoteId, LimitOrderSubmissionMetadata>` currently, and trying to access `orderSubmission[quoteId]` in `selectLimitOrderSubmissionMetadata` will be typed as `LimitOrderSubmissionMetadata`, however, the only reason why it was always defined was sheer luck.

In reality, we may well do undefined accessors, and that's precisely what's happening now when trying to refresh the /limit/confirm route without limit data in the store. 

Ideally we'd type the whole thing as a `PartialRecord<T, U>` but don't want to make the diff too boilerplaty for that reason, as this will require narrowing things in the whole `limitOrderSlice` reducer, TBD in a follow-up!